src/replica/replica.cpp

```diff
@@ -421,6 +429,7 @@ void replica::close()
 {
     dassert_replica(status() == partition_status::PS_ERROR ||
                         status() == partition_status::PS_INACTIVE ||
+                        _disk_migrator->status() == disk_migration_status::IDLE ||
```

Comment:
Will duplication follower affect disk migrator status?

Reply:
Here is a bug and can't find in previous, added unit test won't pass when test `close`